### ISSUE TITLE
Export/import publisher/resource metadata

### ISSUE THREAD

**Issue by tigreped** [WingLongitude/lontra-harvester#38]
Currently, harvesting data is input manually. There is no way to export or import this data to/from SQL. Implement in order to provide a proper method of avoiding rework.

**Comment by cgendreau** [comment]
Why would you export/import publisher?

**Comment by tigreped** [comment]
As a backup measure. So far, we need to enter publisher/resource
information manually. If there is any problem with the database, currently,
we already would have to reenter 91 publishers and 160 resources.

Once  this process is automated in the indexer to fetch this info from the
IPT url, for instance, this will still be useful to export/import the list
of IPTs to be managed.

**Comment by cgendreau** [comment]
Why not simply using a `pg_dump -t ` ?

**Comment by tigreped** [comment]
I'm always keeping the angle that there should be user interfaces that are
friendly enough so that non IT people can manage harvesting. The way I see
it, that eliminates the dump possibility. What do you think?

**Comment by cgendreau** [comment]
The lontra-harvester should be considered an expert tool that requires documentation and an IT person to at least install and updating it. I don't think a such feature should be implemented unless all the other issues are solved. I would rather document the procedure to backup and restore the tables.

**Comment by tigreped** [comment]
I agree it should require an IT person to install and update it (as the versions improve, right?). I also agree we can document  the procedure to backup and restore these basic tables. I also agree we should solve other issues first, but I don't see how that removes the importance of this issue. If the issue makes sense, despite of other priorities, I suppose we should keep it open, right?
